packages/opentelemetry/src/index.ts

```diff
@@ -202,11 +227,10 @@ export async function init(config: OpenTelemetryConfig) {
     );
   }
 
-  const tracerProvider = new NodeTracerProvider({
+  tracerProvider = new NodeTracerProvider({
```

Comment:
I think this was a bug 🙃 without this, `tracerProvider` would always be falsy in `shutdown()`.

Reply:
Yes! Makes sense.